app/controllers/users/passwords_controller.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+module Users
+  class PasswordsController < Devise::PasswordsController
+    layout 'blank'
+  end
+end
```

Comment:
Add an empty line at the end of the files. You can configure your vs code to do it

Reply:
I already configured it and in my local file the empty line actually appears, why could it be?

Reply:
Do you have this on your settings.json ?
```ruby
  "editor.formatOnSave": true,
  "files.insertFinalNewline": true,
  ```

Reply:
this file has the empty line at the end. The issue is that Github does not render them. When the empty line in the end of file is missing, github shows a 🚫 sign at the end of the file.